skimage/feature/texture.py

```diff
@@ -74,8 +74,10 @@ def graycomatrix(image, distances, angles, levels=None, symmetric=False, normed=
 
     Examples
     --------
-    Compute 2 GLCMs: One for a 1-pixel offset to the right, and one
-    for a 1-pixel offset upwards.
+    Compute 4 GLCMs using 1-pixel distance and 4 different angles. For example,
+    an angle of 0 radians refers to the neighboring pixel to the right;
```

Comment:
```suggestion
    an angle of 0 radian refers to the neighboring pixel to the right;
```

Reply:
I used "radians" intentionally since that's the convention I found on https://en.wikipedia.org/wiki/Radian. My logic is that anything that's not precisely 1 radian, should be plural. But I don't feel strongly at all about this. 

I'll leave the decision to you since you are often more informed on things like this. :wink:

Reply:
Oh, thanks for the reference (although I definitely don't consider Wikipedia to be the *ultimate* reference)! I have quickly searched the page but couldn't find any instance of "0 radians" nor "zero radians" though...

The logic/grammar is that the plural applies to quantities > 1 (that's what I remember learning in fourth grade, but we could probably find written references).

Reply:
I'd say that's the more common form in English: 5 degrees Celcius, 2 radians, etc., even though my physics professor hated it :sweat_smile: